mysite/static/js/account/edit_photo.js

```diff
@@ -0,0 +1,64 @@
+var csrf_token = null;
+var width = null;
+
+function set_callback_for_image_upload(token){
+    $("#edit_photo").on("submit", upload_image);
+    csrf_token = token;
+    width = $("#progressBar").width();
+    $('#clear_message').click(hide_notification);
```

Comment:
You're using `.on()` above but `.click()` here. I've seen some schools of thought that suggest keeping a consistency of jQuery calls by doing `.on('click', ...` rather than `.click()` 

The idea is that `.on('click')` reads better to future coders. The way things are now, I read this code and thought "Why are we forcing a click on that element?" then remembered what .click actually does.

The jQuery docs indicate that `.on()` wasn't added (or was last added) in jQuery 1.7, and best I can tell OH is running 1.3. I'd love to see more testing in more browsers to make sure the behavior is as-expected.